File: Vepr-bot/models/productphotos.py
class Productphotos:

	def __init__(self):
		self.product = 0
		self.url_photo = " "
		self.photo_name = " "
		self.id = 0

	@property
	def _product(self):
		return self.product

	@_product.setter
	def _product(self, value):
		self.product = value



	@property
	def _url_photo(self):
		return self.url_photo

	@_url_photo.setter
	def _url_photo(self, value):
		self.url_photo = value

	@property
	def _photo_name(self):
		return self.photo_name

	@_photo_name.setter
	def _photo_name(self, value):
		self.photo_name = value



	@property
	def _id(self):
		return self.id

	@_id.setter
	def _id(self, value):
		self.id = value


	def get(self):
		productphotosDict = self.__dict__
		del productphotosDict['id']
		productphotosDict['idProduct'] = productphotosDict.pop('product')
		return productphotosDict

	def set(self, product, url_photo, id, photo_name):
		self.product = product
		self.url_photo = url_photo
		self.photo_name = photo_name
		self.id = id
		return self

File: vepr_backend/lib/model/product_photos.dart
import 'package:conduit/conduit.dart';
import 'package:vepr_backend/model/product.dart';

class ProductPhotos extends ManagedObject<_Product_Photos> implements _Product_Photos {}

class _Product_Photos{
  @primaryKey
  int? id;
  @Column(indexed: true, useSnakeCaseName: true, nullable: true)
  String? urlPhoto;
  @Column(indexed: true, useSnakeCaseName: true, nullable: true)
  String? photoName;

  @Serialize(input: true, output: false)
  int? idProduct;
  
  @Relate(#productPhotosList, isRequired: true, onDelete: DeleteRule.cascade)
  Product? product;

}